Cargo.toml

```diff
@@ -44,3 +44,4 @@ codegen-units = 1
 
 [package.metadata.docs.rs]
 all-features = true
+rustdoc-args = ["--cfg", "docsrs", "--cfg", "feature=\"backend-mmap\"", "--cfg", "feature=\"backend-atomic\""]
```

Comment:
M understanding is that `rustdoc-args` enables features in the rustdoc binary, so this only needs to be `rustdoc-args = ["--cfg", "docsrs"]`. The features `backend-mmap` etc. will be enabled by `all-features = true`.

Reply:
I've seen the later commit removes these, I think in this case it would make sense to squash the commits in this PR.

Reply:
Wait, we don't get paid by the commit? 

Squashed it. Thanks for the patience.